frontend/index.html

```diff
@@ -531,15 +527,56 @@
         // we would try to load ntrips_purpose__default.png
         const imgFile = "plots/" + metric + "_" + dateVal + program + ".png";
+        const htmlFile = "plots/" + metric + "_" + dateVal + program + ".html";
         const altTextFile = "plots/" + metric + "_" + dateVal + program + ".txt";
         const altText = loadFile(altTextFile);
-        const nw = ["<li data-min-sizex='2' data-min-sizey='2'>"
+        const isStackedMetric = ['ntrips_total', 'ntrips_purpose', 'ntrips_under80', 'ntrips_commute_mode_confirm', 'total_trip_length'].includes(metric);
+        const jsonData = { metric, dateVal, program, metricLabel, dateLabel, programLabel, sizex, sizey };
+
+        if (isStackedMetric){
+          const nw = ["<li data-min-sizex='2' data-min-sizey='2'>"
+            + labelText
+            + "&nbsp;"
+            + "<button class= 'btn btn_default addInfo' style='color: black; background-color: none; border-color: white;' data-info='" + encodeURIComponent(JSON.stringify(jsonData))+ "'> More Info </button>"
+            + "&nbsp;"
+            + "<button class='remove'>x</button>"
+            + "<img src='" + imgFile + "' alt='" + altText + "' width=90% height=90%>"
+            + "</li>",
+            sizex, sizey];
+          gridster.add_widget.apply(gridster, nw);
+        }
+        else{
+          const nw = ["<li data-min-sizex='2' data-min-sizey='2'>"
+            + labelText
+            + "<button class='remove'>x</button>"
+            + "<img src='" + imgFile + "' alt='" + altText + "' width=90% height=90%>"
+            + "</li>",
+            sizex, sizey];
+          gridster.add_widget.apply(gridster, nw);
+        }
+      });
+
+      $('body').on("click", ".gridster ul > li .addInfo", function () {
+        const addButton = $(this);
+        const additionalInfo = JSON.parse(decodeURIComponent(addButton.data('info')));
+        const metric = additionalInfo.metric;
+        const dateVal = additionalInfo.dateVal;
+        const program = additionalInfo.program;
+        const metricLabel = additionalInfo.metricLabel;
+        const dateLabel = additionalInfo.dateLabel;
+        const programLabel = additionalInfo.programLabel;
+        const sizex = additionalInfo.sizex;
+        const sizey = additionalInfo.sizey;
+        const labelText = metricLabel + " " + dateLabel + " " + programLabel;
+        const htmlFile = "plots/" + metric + "_" + dateVal + program + ".html";
+
+        const nw_additionalInformation = ["<li data-min-sizex='2' data-min-sizey='2'>"
           + labelText
           + "<button class='remove'>x</button>"
-          + "<img src='" + imgFile + "' alt='" + altText + "' width=90% height=90%>"
+          + "<object data='" + htmlFile + "' type='text/html' style='margin-top: 20px;' width='90%' height='90%'></object>"
           + "</li>",
           sizex, sizey];
-        gridster.add_widget.apply(gridster, nw);
+        gridster.add_widget.apply(gridster, nw_additionalInformation);
```

Comment:
I think this raises a larger decision about if this HTML feature is a general "drill-down" feature, or if this is something we are trying to optimize towards accessibility. 

If lumping small %s together, the table functions to "explode" the other and therefore it would absolutely be useful to see the chart and the table at the same time (which toggling would prevent).

However, the new grid element does just show up ~wherever it fits~ on the screen, it might be hard to navigate to with a screen reader. If we want to make an effort to center accessibility here, I would advocate for testing the current implementation with a screen reader and consider the toggle if the experience is bad.

Also, if the toggle is relatively fast - people can click back and forth between the two to compare if wanted, maybe this would feel a little bit like a tooltip/hover interaction?

Reply:
We have decided not to incorporate any changes related to this in this PR.